index.js

```diff
@@ -187,6 +187,9 @@ function use (extensions) {
     this.on('close', onfinalize)
     this.on('end', onfinalize)
     this.on('finish', this.finalize)
+    this.on('error', function (err) {
+      debug('stream error', err)
+    })
```

Comment:
gah yeah that's so hard to tell with node. A swallowed error event is way worse than one that crashes the system..

Just to debug, you could find out what is listening like this

```js
this.on('error', function (err) {
  this.listeners('error').forEach(function (fn) {
    console.log('FN', fn.toString())
  })
})
```